Data-Structures/src/main/java/stacksandqueues/Stack.java

```diff
@@ -0,0 +1,40 @@
+package stacksandqueues;
+
+public class Stack<T> {
+  Node<T> top;
+
+  public Stack() {
+  }
+
+  public void push(T value) {
+    Node <T> temp;
+    if (top == null){
+      top = new Node(value, null);
+    }
+    else {
```

Comment:
For each other `else` in the code the `else` is on the same line as the `if`'s closing `}`. Striving for consistency will help make your code as easy to read as possible